functions/src/create-contract.ts

```diff
@@ -133,6 +133,26 @@ export const createmarket = newEndpoint({}, async (req, auth) => {
   if (ante > user.balance)
     throw new APIError(400, `Balance must be at least ${ante}.`)
 
+  let group = null
```

Comment:
Should this be typed better? `let group: Group | null`

Reply:
Yes, I guess it should.